src/audio/kpb.c

```diff
@@ -308,9 +312,9 @@ static int kpb_bind(struct comp_dev *dev, void *data)
 
 		if (sink_buf_id == buf_id) {
```

Comment:
It seems this could be simplified to 
```suggestion
		if (sink_buf_id == buf_id)
		        kpb->sinks[bu->extension.r.src_queue] = sink;
```